legend-pure-ide-light/src/main/java/org/finos/legend/pure/ide/light/session/PureSession.java

```diff
@@ -85,10 +84,27 @@ private FunctionExecution initialize()
 
         this.functionExecution = new FunctionExecutionInterpreted(VoidExecutionActivityListener.VOID_EXECUTION_ACTIVITY_LISTENER);
 
+        for(String property : System.getProperties().stringPropertyNames())
```

Comment:
Is not all this available already using `RuntimeOptions.systemPropertyOptions("pure.option.")`

That seems to avoid the need of extra methods, and parsing, etc.

Reply:
Good point - I hadn't spotted this method.

However I've updated the PR to add an API to toggle options - the intention is that the state of runtime options is kept in PureSession.
At startup we read any options set by system properties and then the options can be toggled at runtime without having to restart the IDE.

This API could also be used by the frontend so I'm exposing it now and then the frontend can be updated later to take advantage of this.